dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GitHubUtil.java

```diff
@@ -68,9 +68,8 @@ public GHRepository createFork(GHRepository repo) throws IOException {
         try {
             return repo.fork();
         } catch (IOException e) {
-            log.error("Could not fork {}", repo.getFullName(), e);
+            throw new IOException(String.format("Unable to fork %s", repo.getFullName()), e);
         }
-        return null;
```

Comment:
Was anything using this? This seems like a behavioral change.

Is this an exceptional case?

Reply:
Returned value is not used anywhere.

Reply:
Any reason you couldn't just check to see if the return value is null from this rather than using exceptions for flow control?

Reply:
Modified the flow to check for return value.